tests/test-quickstart/src/test/java/org/eclipse/jetty/quickstart/PreconfigureSpecWar.java

```diff
@@ -52,10 +51,12 @@ public static void main(String[] args) throws Exception
 
         LOG.info("Preconfigured in {}ms", NanoTime.millisSince(__start));
 
+        /*
```

Comment:
Why is this commented out?

Reply:
It just output the quickstart-web.xml to System.out, a useful thing when debugging, but noisy otherwise.

If you need to see it during debugging, then just comment it out.